packages/design-system/src/components/OcBreadcrumb/types.ts

```diff
@@ -1,8 +1,10 @@
 import { RouteLocationRaw } from 'vue-router'
 
 export interface BreadcrumbItem {
+  id?: string
   text: string
   to?: RouteLocationRaw
   allowContextActions?: boolean
   onClick?: () => void
+  isPreviousHiddenFolder?: boolean
```

Comment:
IsPrevioudFolderHidden ?

Reply:
@JanAckermann its the name for the "...", don't know a better name

Reply:
isPreviousHiddenFolder -> IsPrevioudFolderHidden i tought